EIPS/eip-7745.md

```diff
@@ -0,0 +1,57 @@
+---
+eip: 7745
```

Comment:
```suggestion
eip: 7735
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.